internal/controlplane/handlers_evalstatus.go

```diff
@@ -278,6 +278,14 @@ func buildRuleEvaluationStatusFromDBEvaluation(
 			Msg("error converting severity will use defaults")
 	}
 
+	entityInfo := map[string]string{}
+	if eval.Entity == db.EntitiesPullRequest {
```

Comment:
Was this supposed to be `db.EntitiesPullRequest` or repository?

Reply:
Awesome catch, fixed.